server/src/main/twirl/views/tags/_asset.scala.html

```diff
@@ -1 +1,2 @@
-@(path: String)(implicit config: play.api.Configuration)@{config.getString("application.cdn").getOrElse("") + routes.Assets.versioned(path)}
\ No newline at end of file
+@import drt.server.feeds.Implicits.maybeOperatorFromString
```

Comment:
This import shouldn't be here - it's used in feed code to add an airline operator